modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/date/date_field.js

```diff
@@ -93,12 +116,16 @@ AUI.add(
 									dateMask.push(/\d/, /\d/);
 								}
 
-								if (index < (items.length - 1)) {
-									dateMask.push('/');
+								if (index < 2) {
```

Comment:
Is there a reason why you changed this from being based on the length of items to a hardcoded value?

Reply:
Majority of the locales use 2 delimiters which split into some ordering of month, date, and year, but some locales have 3 delimiters which split into an ordering of month, date, and year, plus a blank. Using 2 prevents us from adding in an extra blank.